components/v1/swr/useLeveragedTokenNAV.ts

```diff
@@ -21,14 +21,16 @@ const LeveragedTokenNAVFetcher = async (args: LeveragedTokenNAVFetcherArgs): Pro
 };
 
 // Get the latest NAV of the leveraged token
-export function useLeveragedTokenNAV(req: LeveragedTokenNAVRequest) {
-    const { data, error } = useSWR<ethers.BigNumber, Error>(
+export function useLeveragedTokenNAV(req: LeveragedTokenNAVRequest, options?: SWRConfiguration) {
+    const { data, error, mutate } = useSWR<ethers.BigNumber, Error>(
```

Comment:
Why we need `mutate` ?

Reply:
because i want to re-fetch data when error appears

Reply:
because of its specific usage. 
this effect only runs when dependencies change.
because I think if you combine them, it may create unexpected re-render and side effect